ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -316,8 +316,28 @@ extension SignalProducerType {
 	///
 	/// Returns a Disposable which can be used to interrupt the work associated
 	/// with the Signal, and prevent any future callbacks from being invoked.
-	public func start(error error: (E -> ())? = nil, completed: (() -> ())? = nil, interrupted: (() -> ())? = nil, next: (T -> ())? = nil) -> Disposable {
-		return start(Event.sink(next: next, error: error, completed: completed, interrupted: interrupted))
+	public func startWithNext(error error: (E -> ())? = nil, completed: (() -> ())? = nil, interrupted: (() -> ())? = nil, next: (T -> ())? = nil) -> Disposable {
```

Comment:
Although, I would sacrifice the trailing closure syntax to require the associated closure be the first param and required.

```swift
// this still seems reasonable to me ...
producer.startWithNext({
    println("Next \($0)")
})
```